notebook/notebookapp.py

```diff
@@ -113,6 +114,7 @@
 from ._tz import utcnow, utcfromtimestamp
 from .utils import (
     check_pid,
+    enable_json_logs,
```

Comment:
This can go away now, we only need to read the env var from the default function.